src/panels/profile/ha-refresh-tokens-card.ts

```diff
@@ -24,6 +24,21 @@ import {
 import { haStyle } from "../../resources/styles";
 import { HomeAssistant } from "../../types";
 
+const compareTokenLastUsedAt = (tokenA: RefreshToken, tokenB: RefreshToken) => {
+  const timeA = tokenA.last_used_at;
+  const timeB =  tokenB.last_used_at;
+  if (timeA === undefined) {
+    return 1;
+  } else if (timeB === undefined) {
+    return -1;
+  } else if (new Date(timeA) < new Date(timeB)) {
+    return 1;
+  } else if (timeA > timeB) {
+    return -1;
+  }
+  return 0;
+}
```

Comment:
```suggestion
const compareTokenLastUsedAt = (tokenA: RefreshToken, tokenB: RefreshToken) => {
  const timeA = tokenA.last_used_at ? new Date(timeA) : 0;
  const timeB =  tokenB.last_used_at ? new Date(timeB) : 0;
  if (timeA < timeB) {
    return 1;
  } else if (timeA > timeB) {
    return -1;
  }
  return 0;
}
```

Reply:
oh duh🤦. I totally forgot about using a ternary operator when setting the const. Thanks!